extensions/amp-story/validator-amp-story.protoascii

```diff
@@ -267,6 +267,10 @@ tags: {
     name: "translate-y"
     value_regex_casei: "[0-9]+px"
   }
+  attrs: {
+    name: "pan-static-scale"
+    value_regex_casei: "[0-9]+([.][0-9]+)?"
```

Comment:
Hi, I'll review the validation changes on behalf of wg-caching group.

Should the regex also allow a decimal point with no fractional part or no integer part, e.g. "123." and ".2"? E.g. css scale transform allows both (but not both at the same time).

Reply:
No problem, done.